### ISSUE TITLE
reverse the slicing directions

### ISSUE THREAD

**Issue by Gilbert931** [adenzu/Manga-Panel-Extractor#12]
I apologize for the stupid question but is it possible to somehow set the panel slicing in the order: from left to right and from TOP to bottom?

**Comment by adenzu** [comment]
Currently there is no option to specify that. I do not understand why did you ask this

**Comment by Gilbert931** [comment]
Thought there was an option to get in the right sort. Classic manga reading, left to right, top to bottom like on my screenshot

![image](https://github.com/user-attachments/assets/7f6c2b73-8c83-48ac-a895-2ccb86658e56)

**Comment by Gilbert931** [comment]
By the way, I made a mistake. The manga reads from right to left (but it's still top to bottom). I wrote left to right out of habit :D Sorry for that

Very much like your work) Tested all day, the site works very fast and stable

**Comment by adenzu** [comment]
Oh so you were talking about the website. There is also a desktop app version which might be useful for your case.

The website currently lacks in functionality compared to the desktop app, this is because I'm less knowledgeable when it comes to developing websites.

If you want to check out the desktop app, you can find it [here](https://github.com/adenzu/Manga-Panel-Extractor/releases/latest).

**Comment by Gilbert931** [comment]
No, I was talking about the desktop app)) Just to say that it's a very cool project, website included

I actively use the program. There are minor flaws: pictures are sometimes duplicated (for example, if the program slices a large picture, it can slice a small one, which is part of a large one. It turns out a kind of duplicate content). Also the output pictures in a strange order come out, I already said that)

Do not take as a serious criticism, just a subjective opinion of what is missing, perhaps useful to you in refining the program :)

**Comment by adenzu** [comment]
Yeah the issues you mentioned are indeed current flaws of the program, sorry for that. I would like fix those, and I know for a fact the app can be better. I have seen very good similar projects utilizing AI to do this job, right now my project does not AI and it could benefit a lot from it.

I hope and plan to provide you and other users a better service in time by incorporating such tools. As of now this Issue should stay here for me and others to see. Thanks for writing it!
